CHMeetupApp/Sources/ViewControllers/Events/EventPreview/EventPreviewViewController.swift

```diff
@@ -82,14 +82,14 @@ class EventPreviewViewController: UIViewController {
     }
   }
 
-  func acceptAction() {
+  @objc func acceptAction() {
```

Comment:
Use line break for function, when you set attributes, for example, @objc or @discardableresult

Reply:
maan)